bin/detect

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [ -f package.json ]; then
+  echo "node-fn-structured-cloudevent"
```

Comment:
Is this something we should be doing in CNB buildpacks? I put this here because I felt like it was a pattern in legacy heroku buildpacks, but CNB's lifecycle kind of prints this for you now.

Reply:
Yea, should do this. It will show up in the debug logs for `pack build`